app/components/article-featured-video.js

```diff
@@ -33,16 +33,16 @@ export default Component.extend(
 		 * @returns {void}
 		 */
 		initOnScrollBehaviour() {
-			var $video = this.$('.video-container'),
+			let $video = this.$('.video-container'),
 				$siteHead = $('.site-head'),
 				videoBottomPosition = $video.offset().top + $video.height(),
 				showVideoOnScroll = true;
 
-			$(window).on('scroll', { player: this.player, $window: this.$(window) }, function({data: {player, $window}}) {
+			$(window).on('scroll', {player: this.player, $window: this.$(window)}, function ({data: {player, $window}}) {
```

Comment:
As @rogatty mentioned before. Please remember about unbinding this event when video player is destroyed. Currently it'll stay active on the next pageview and can have unexpected consequences when there is no video.